openmmtools/utils.py

```diff
@@ -219,6 +225,15 @@ def _math_eval(node):
 # QUANTITY UTILITIES
 # =============================================================================
 
+# List of simtk.unit methods that are actually units and functions instead of base classes
+# Pre-computed to reduce run-time cost
+# Get the built-in units
+_VALID_UNITS = {method: getattr(unit, method) for method in dir(unit) if type(getattr(unit, method)) is unit.Unit}
+# Get the built in unit functions and make sure they are not just types
+_VALID_UNIT_FUNCTIONS = {method: getattr(unit, method) for method in dir(unit)
```

Comment:
If it can simplify your life, you could also make use of `inspect.getmembers(unit, predicate=inspect.isfunction)`.